config/index.js

```diff
@@ -1,6 +1,12 @@
 /* eslint no-process-env: "off" */
 require('dotenv').config()
 
+const API_VERSION = process.env.API_VERSION
+
+if (!API_VERSION) {
+  throw new Error('API_VERSION must be set')
```

Comment:
Maybe add in that its the environment variable that needs setting so its clearer from the error message: `The `API_VERSION` environment variable must be set`?

Reply:
👍